File: lc287.java
import java.util.Arrays;

public class lc287 {
//    287. Find the Duplicate Number
    public static void main(String[] args) {
        int nums[]={7,9,7,4,2,8,7,7,1,5};
        Arrays.sort(nums);
        int ans=0;
        for (int i=0;i<nums.length-1;i++){
            if (nums[i]==nums[i+1]){
                ans=nums[i];
                break;
            }
        }
        System.out.println(ans);
    }
}

File: lc287.py
from typing import List


class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        nums.sort()
        ans=0
        for i in range(len(nums)-1):
            if nums[i] == nums[i+1]:
                ans=nums[i]
                break
        return ans
print(Solution().findDuplicate([1,2,3,4,5,6,7,7]))